pkg/apis/kubeone/helpers.go

```diff
@@ -35,11 +35,7 @@ const (
 	credentialSecretName = "kube-system/kubeone-registry-credentials" //nolint:gosec
 )
 
-var (
-	v124Constraint         = semverutil.MustParseConstraint(">= 1.24.0, < 1.25.0")
-	v125Constraint         = semverutil.MustParseConstraint(">= 1.25.0, < 1.26.0")
-	v126AndNewerConstraint = semverutil.MustParseConstraint(">= 1.26.0")
-)
+var olderThanV131Constraint = semverutil.MustParseConstraint("< 1.31")
```

Comment:
This might just be me, but my brain read "older than" as ">" at first.

Maybe rename to `preV131Constraint`?

Reply:
I'm struggling to come up with a better name, let's go with `preV131Constraint` as of now. Hopefully we can completely remove this variable at some point soon.